src/component/mxgraph/BpmnMxGraph.ts

```diff
@@ -104,28 +102,28 @@ export class BpmnMxGraph extends mxGraph {
 
   createMouseWheelZoomExperience(config: ZoomConfiguration): void {
     config = ensureValidZoomConfiguration(config);
-    mxEvent.addMouseWheelListener(debounce(this.getZoomHandler(true), config.debounceDelay), this.container);
-    mxEvent.addMouseWheelListener(throttle(this.getZoomHandler(false), config.throttleDelay), this.container);
+    mxgraph.mxEvent.addMouseWheelListener(debounce(this.getZoomHandler(true), config.debounceDelay), this.container);
+    mxgraph.mxEvent.addMouseWheelListener(throttle(this.getZoomHandler(false), config.throttleDelay), this.container);
```

Comment:
Here and in lot of places, we could still call directly `mxEvent`.
This would require to declare `const mxEvent = mxgraph.mxEvent`
We cannot do this when a class is also imported for types resolution 
So this means that we would have some inconsistencies at some place: in some files, we instantiate some mxPoint and need it as type in method signatures.

Reply:
We have decided to keep the current form (mxgraph.XXX) everywhere.